Cargo.toml

```diff
@@ -60,6 +60,8 @@ felt = { package = "cairo-felt", path = "./felt", default-features = false, feat
 ] }
 bitvec = { version = "1", default-features = false, features = ["alloc"] }
 
+parity-scale-codec = { version = "3.6.0", no-default-features = true }
```

Comment:
it's not the same version we use in Madara, can we try to uniform and use same versions. Fine to upgrade Madara to this version if it works

Reply:
cargo handles it automatically but yes we should update the madara version